RUFAS/routines/animal/manure/dry_cow_manure_excretion.py

```diff
@@ -6,96 +6,154 @@
 from RUFAS.routines.animal.ration.ration_driver import ration_report
-import math
 
 
-def manure_calculations(ration_formulation, feed, bw, milk_prod, p_feces_excrt,
-                        p_urine, ME_intake):
-    """
-    TEMPORARY PLACEHOLDER
-    Calculates inputs for manure module with information from the
-    ration formulation. Equations referenced are from pseudocode.
-
-    Args:
-        ration_formulation: dictionary which stores the calculated ration
-        feed: instance of the Feed class
-        bw: body weight, kg
-        milk_prod: milk production, kg
-        p_feces_excrt: amount of P excreted by an animal (g)
-        p_urine: amount of P required for urine production (g)
-        ME_intake: metabolizable energy intake, Mcal/kg DM
-
-
-    Returns:
-        p_excrt: amount of P excreted by animal, g
-        and a dictionary containing the following values
-            U: urea concentration, mol/L
-            Urine: # TODO: Add description
-            TAN_s: total ammoniacal nitrogen concentration in the manure slurry,
-                mol/L
-            MN: nitrogen in liquid and solid manure, g
-            Mkg: amount of manure, kg
-            VSd: degradable volatile solids, g
-            VSnd: non-degradable volatile solids, g
-            WIP_frac: water extractable inorganic P fraction
-            WOP_frac: water extractable organic P fraction
-            p_excrt_manure: manure P excretion for manure module input (g)
-            p_frac: P fraction of manure
-            K: potassium in manure, g/day
-
-
-
-    Further calculations to account for entire diet:
-    DMI: dry matter intake, kg
-    DM: dietary dry matter, % of diet
-    CP: dietary crude protein, % of DM
+def manure_calculations(ration_formulation,
+                        feed,
+                        body_weight: float,
+                        daily_milk_production: float,
+                        fecal_phosphorus: float,
+                        urine_phosphorus_required: float,
+                        metabolizable_energy_intake: float) \
+        -> Tuple[float, AnimalManureExcretions]:
+    """Calculates the manure excretion values for a non-lactating cow with information from the ration formulation.
+
+    Parameters
+    ----------
+    ration_formulation : Dict[str, float]
+        Dictionary that stores the calculated ration.
+    feed : Dict[str, float]
+        A Feed object that contains information about the available feeds.
+    body_weight : float
+        Body weight of the current animal, kg.
+    daily_milk_production : float
+        Daily milk production of the current animal, kg.
+    fecal_phosphorus : float
+        Amount of fecal phosphorus excreted by the current animal, g.
+    urine_phosphorus_required : float
+        Amount of phosphorus required for urine production, g.
+    metabolizable_energy_intake : float
+        Metabolizable energy intake, Mcal/kg dry matter.
+
+    Returns
+    -------
+    float
```

Comment:
does this need the variable name it's returning before the type?
```suggestion
    total_phosphorus_excreted : float
```